pages/13.builders/docs.en.md

```diff
@@ -68,7 +68,82 @@ From the 3.3 General Availability release there will be three email templates th
 
 ### Themes
 
-If you search through the list of available themes, the new MJML themes `Brienz`, `Paprika` and `Confirm Me` will not be available until you enable the GrapesJS plugin. 
+If you search through the list of available themes, the new MJML themes `Brienz`, `Paprika` and `Confirm Me` will not be available until you enable the GrapesJS plugin.
+
+### Creating new Themes
+
+#### Why to use email templates?
+The GrapesJs makes possible to insert HTML code and edit it in a WYSIWYG environment.
+A template can feature a wide variety of blocks and sections reflecting desired email design.
+By choosing a template and removing the unnecessary blocks beautiful emails can be created very efficiently.
+
+#### Modifying Legacy template
+The GrapesJs Builder looks at the template's config file content before a template is displayed for selection.
+As of Mautic 3.3 a new line has to be added, that defines which email designer is the template suitable for:
+
+File name: config.json
+
+`    {
+      "name": "Great Template",
+      "author": "Mr. Robot",
+      "authorUrl": "https://mautic.org",
+      "builder":  ["grapesjsbuilder"],
+      "features": [
+        "email"
+      ]
+    }`
```

Comment:
```suggestion
    }```
```

Reply:
I think we want to use the code block rather than line syntax but the suggestion feature on GitHub doesn't like it as that is what it uses to separate the suggestions.

Here is the syntax to use from the styleguide: https://docs.mautic.org/en/contributing/style-guide#code-blocks (three backticks before and after the code block)